android/libraries/rib-base/src/main/java/com/badoo/ribs/android/IntentCreator.java

```diff
@@ -28,15 +28,4 @@
    * @return The intent.
    */
   Intent create(Class<?> cls);
```

Comment:
What should I do if I want to launch external Activity (outside my app's package)?
Considering Single Activity approach, that will be the most popular case.

Reply:
Added empty intent creation to interface.